tests/integration_tests/document_loaders/test_pdf.py

```diff
@@ -11,7 +11,25 @@
 )
 
 
-def test_unstructured_pdf_loader() -> None:
+def test_unstructured_pdf_loader_elements_mode() -> None:
```

Comment:
This works, but CI won't run these tests if they're not registered as unit-tests.

If you have time, you could move these under the `tests/unit_tests` instead (find the other code that does pdf parsing there) and use pytest.mark.requires(unstructured) marker

This has a bit of instructions on how test optional dependencies: 

https://github.com/hwchase17/langchain/blob/master/.github/CONTRIBUTING.md#working-with-optional-dependencies